### PR TITLE
chore: Sync latest translations from Crowdin

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/i18n/webapp-de.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "index.claim": "Sicherer Messenger für alle.",
   "index.createAccount": "Konto erstellen",
-  "index.createAccountFor": "für Privat",
+  "index.createAccountFor": "für den privaten Gebrauch",
```
Comment (gregor) [inline app/i18n/webapp-de.json]: I asked Roger to take a look at this yesterday. Did the change come from him?

File: app/script/localization/translations/webapp-cs.js [hunk 2]
```diff
@@ -248,8 +248,6 @@ z.string.cs.conversation_context_menu_delete_everyone = 'Odstranit pro všechny'
 z.string.cs.conversation_context_menu_download = 'Stáhnout';
 z.string.cs.conversation_context_menu_like = 'Líbí';
 z.string.cs.conversation_context_menu_unlike = 'Nelíbí';
-z.string.cs.conversation_delete_timestamp = 'Smazáno v {{date}}';
-z.string.cs.conversation_edit_timestamp = 'Upraveno v {{date}}';
```
Comment (gregor) [inline app/script/localization/translations/webapp-cs.js]: Do not delete these here or in other languages.

Comment (AndyLnd) [reply app/script/localization/translations/webapp-cs.js]: I actually didn't delete anything, as those are just the translations I get from crowdin.

File: app/i18n/webapp-de.json (hunk 1 above)
Comment (AndyLnd) [reply app/i18n/webapp-de.json]: I can't tell. We will have to ask him, if he changed anything on crowdin

**Pull request merged**